src/eBayEnterprise/RetailOrderManagement/Payload/Payment/ConfirmFundsReply.php

```diff
@@ -113,7 +118,7 @@ protected function serializeContents()
             $this->serializePaymentContext() :
             $this->serializePaymentContextBase();
         return $paymentContext .
-        "<FundsAvailable>{$this->xmlEncode($this->getFundsAvailable())}</FundsAvailable>";
+        "<FundsAvailable>{$this->xmlEncode($this->getFundsAvailable())}</FundsAvailable>" . "<ReauthorizationAttempted>{$this->xmlEncode($this->getReauthorizationAttempted())</ReauthorizationAttempted>";
```

Comment:
to keep this clean you should return this string value from a protected `serializeReauthorizationAttempted` method